docs/api/rest_api/rest_api_reference/input/examples/ai/actions/GET/ActionConfigurationList.json.example

```diff
@@ -0,0 +1,218 @@
+{
+    "ActionConfigurationList": {
+        "_media-type": "application/vnd.ibexa.api.ai.ActionConfigurationList+json",
+        "_href": "/api/ibexa/v2/ai/actions",
+        "ActionConfiguration": [
+            {
+                "_media-type": "application/vnd.ibexa.api.ai.ActionConfiguration+json",
```

Comment:
I would regenerate this response as there were some changes to names, specifically lowercase attribute names